src/board/Board.jsx

```diff
@@ -1,20 +1,32 @@
+import React, { useState } from "react";
 import Row from "./Row";
 import "./Board.css";
+import { defaultBoardData } from "./boardData";
+import { updateBoard } from "./updateBoard";
 import PropTypes from "prop-types";
-const { arrayOf, number } = PropTypes;
 
-function Board({ data }) {
+function Board({ changeTurn, currentPlayer }) {
+  const [boardData, setBoardData] = useState(defaultBoardData);
+
+  const handleClick = (cellValue) => {
+    const updatedBoard = updateBoard(boardData, cellValue, currentPlayer);
+    setBoardData(updatedBoard);
+
+    changeTurn();
+  };
```

Comment:
This works here, you are using internal state to remember the board and calling the prop when it is done.